bin/rails

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env ruby
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```

Comment:
It might be, but I just followed documentation. 

The thing is that it doesnt actually out put that string, so maybe its needed for something?